addons/hr_recruitment_skills/tests/test_applicant_skills.py

```diff
@@ -493,3 +493,11 @@ def test_duplicate_certifications_in_the_past_are_not_created(self):
         )
         self.assertEqual(len(self.t_applicant.applicant_skill_ids), 6)
         self.assertEqual(len(self.t_applicant.current_applicant_skill_ids), 4)
+
+    def test_job_with_no_skills_and_degree_with_score_zero(self):
+        job = self.env["hr.job"].create({"name": "No Job"})
+        degree = self.env["hr.recruitment.degree"].create({"name": "Degree", "score": 0})
+        job.expected_degree = degree
+        self.t_applicant.job_id = job
+
+        self.assertFalse(self.t_applicant.matching_score)
```

Comment:
```suggestion
        self.assertEqual(self.t_applicant.matching_score, 0)
```
Goot to asserting actual value 0 instead assert False